assets/queries/k8s/tls_connection_certificate_not_setup/test/negative1.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: joaodanielrufino/kubelet
```

Use the `kube-apiserver` docker image
```suggestion
      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```